From 3f3603195b70f9da684a67f1d215401cfa75e976 Mon Sep 17 00:00:00 2001
From: Fugang Duan <b38611@freescale.com>
Date: Wed, 2 Mar 2016 10:17:02 +0800
Subject: [PATCH 0968/5242] MLK-12480 dts: imx7d-12x12-lpddr3-arm2: remove
 extended enet dts file

commit  c37721ec437df439b3cbc1cc595c7b5819073735 from
https://source.codeaurora.org/external/imx/linux-imx.git

There have two same extended enet dts file to enable fec2 port, so
remove the redundant enet dts file.

The issue is caused by the commit 370426c2a918 that was cherry picked
after commit b74c6b9c7fdc.

Signed-off-by: Fugang Duan <B38611@freescale.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm/boot/dts/imx7d-12x12-lpddr3-arm2-enet.dts |   17 -----------------
 1 file changed, 17 deletions(-)
 delete mode 100644 arch/arm/boot/dts/imx7d-12x12-lpddr3-arm2-enet.dts

diff --git a/arch/arm/boot/dts/imx7d-12x12-lpddr3-arm2-enet.dts b/arch/arm/boot/dts/imx7d-12x12-lpddr3-arm2-enet.dts
deleted file mode 100644
index 151853d..0000000
--- a/arch/arm/boot/dts/imx7d-12x12-lpddr3-arm2-enet.dts
+++ /dev/null
@@ -1,17 +0,0 @@
-/*
- * Copyright (C) 2015 Freescale Semiconductor, Inc.
- *
- * This program is free software; you can redistribute it and/or modify
- * it under the terms of the GNU General Public License version 2 as
- * published by the Free Software Foundation.
- */
-
-#include "imx7d-12x12-lpddr3-arm2.dts"
-
-&epdc {
-        status = "disabled";
-};
-
-&fec2 {
-	status = "okay";
-};
-- 
1.7.9.5

